main.js

```diff
@@ -98,15 +102,91 @@ function deleteNote(e) {
 		if (themeId === notes[i].id) {
 			// console.log(notes[i]);
 			notes.splice(i, 1);
-			console.log(notes);
+			// console.log(notes);
 			localStorage.setItem('notes', JSON.stringify(notes));
 			break;
 		}
 	}
-	console.log(notes);
+	// console.log(notes);
 	render();
 }
 
+// update note
+function renderUpdateBtn() {
+	const updateBtn = document.querySelectorAll('.updateBtn');
+	console.log(updateBtn);
+	for (let i = 0; i < updateBtn.length; i++) {
+		updateBtn[i].addEventListener('click', (e) => updateNoteBtn(e));
+	}
+}
+
+function updateNoteBtn(e) {
+	let themeId = e.target.id;
+	console.log(themeId);
+
+	for (let i = 0; i < notes.length; i++) {
+		if (themeId === notes[i].id) {
+			console.log(notes[i]);
+
+			localStorage.setItem('updateNoteData', JSON.stringify(notes[i]));
+			window.location.href = './update.html';
+			break;
+		}
+	}
+}
+
+updateRenderNote();
```

Comment:
What is this for?

Reply:
> What is this for?
It is a code to call updateRenderNote();.
updateRenderNote() were not worked in other functions 🤔